docs/sce_file_contents.md

```diff
@@ -106,7 +106,10 @@ expt_metadata <- metadata(sce)
 | `usa_mode`          | Boolean indicating whether quantification was done using `alevin-fry` USA mode                                                 |
 | `af_num_cells`      | Number of cells reported by `alevin-fry`                                                                                       |
 | `tech_version`      | A string indicating the technology and version used for the single-cell library, such as 10Xv2, 10Xv3, or 10Xv3.1              |
+| `assay_ontology_term_id` | A string indicating the [Experimental Factor Ontology](https://www.ebi.ac.uk/ols/ontologies/efo) term id associated with the `tech_version`  |
```

Comment:
I was going to comment here to suggest that we link to obofoundry for consistency, only to find that doesn't have EFO. So this is fine. 

(I was then going to suggest linking directly to OLS4 to avoid warnings, but I see that this is scheduled to be replaced at the end of next month, so we are probably fine)